Rsk.Samples.IdentityServer4.AdminUiIntegration/Models/SendResetMfaOneTimeLinkResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Rsk.Samples.IdentityServer4.AdminUiIntegration.Models
+{
+    public class SendResetMfaOneTimeLinkResult
```

Comment:
**nit:** don't forget the private constructor. Otherwise, your factory methods can be bypassed